Makefile

```diff
@@ -25,4 +25,8 @@ docs:
 	@cd docs && make html
 
 requirements:
-	@poetry show --no-dev | tr -s " " | sed 's/ /==/' | sed 's/ .*//' > requirements.txt
\ No newline at end of file
+	@poetry update
```

Comment:
Is doing an implicit poetry update when you are locking the requirements.txt file a good idea, could risk introducing new versions when you go to deploy?

Reply:
Hey @nmunro 👋 
@Hellseher I'm with @nmunro on this one could we get it moved to a separate command in the makefile?
```suggestion
```

Reply:
@mijho ok, it could be removed from `requirements` target and I'm not sure where is better to add it or whether it's required at all. The reason I've added it under `requirements` poetry complains that your dependencies are not up to date and suggested to update them.